src/scenes/Projects/ProjectList.tsx

```diff
@@ -43,11 +40,17 @@ export const ProjectList: FC = () => {
         {data?.projects.total} Projects
       </Typography>
       <Grid container direction="column" spacing={2}>
-        {data?.projects.items.map((item) => (
-          <Grid item key={item.id}>
-            <ProjectListItemCard {...item} />
-          </Grid>
-        ))}
+        {data
+          ? data?.projects.items.map((item) => (
+              <Grid item key={item.id}>
+                <ProjectListItemCard loading={loading} {...item} />
+              </Grid>
+            ))
+          : _.times(5).map((index) => (
```

Comment:
This is weird. Either we are loading or we are not (at least until we get into infinite scroll). The condition should be on `loading` not whether we have data.

Reply:
This was my fault.